src/sentry/models/rule.py

```diff
@@ -27,12 +27,14 @@
 class RuleSource(IntEnum):
     ISSUE = 0
     CRON_MONITOR = 1
+    UPTIME = 2
```

Comment:
We don't need this.

This `RuleSource` thing was a crons idiosyncrasy because we create "hidden" alert rules for crons.

To be clear, from a product perspective, crons and uptime both "appear" to be alert rules (which has nothing to do with this code). But they Are NOT "rules"